APIs/openEO/large_scale_processing.qmd

```diff
@@ -4,14 +4,13 @@ aliases:
+Processing larger areas, especially globally, presents significant challenges in earth observation. Nonetheless, this platform aims to address these challenges. In this context, we highlight one of the best practices by showcasing the example of [processing a croptype map for all 27 countries in the European Union](https://github.com/openEOPlatform/openeo-classification){target="_blank"}. We encourage users to contact the forum with their issues connected to a specific case.
 
 ![EU27 croptype map, processed on openEO](_images/EU27croptypemap.png)
 
-The approach described here is based on local files to track the production. This is a low-cost approach that does not require special IT knowledge but comes with some risks, such as losing your local files. A more robust approach for production-grade projects would typically rely on some sort of database or STAC catalogue service to monitor processing. Such a setup is, however quite similar in many aspects.
+The approach described here relies on local files to track production, offering a low-cost solution. However, this method comes with risks, such as the potential loss of local files. A more robust approach for production-grade projects would typically use a database or STAC catalogue service to monitor processing. However, the setup is quite similar in many aspects.
 
-The basic strategy for processing large areas is to split them up into smaller areas, usually according to a regular tile grid. Splitting reduces the size of the area that needs to be processed by one batch job and avoids running into all kinds of limitations. For instance, when processing a specific projection, you anyway have to stay within the bounds of that projection. Also, the output file size of a job often becomes impractical when working over huge areas. Or you will hit bottlenecks in the backend implementation that does not occur for normally sized jobs. Also, when a smaller job fails or requires reprocessing, the cost will be smaller.
+The basic strategy for processing large areas involves splitting them into smaller sections, usually according to a regular tile grid. This division reduces the area size that needs to be processed in one batch job and avoids various limitations. For example, when processing within a specific projection, it's necessary to stay within the bounds of that projection. Additionally, the output file size of a job can become impractical when dealing with large areas. Bottlenecks that do not occur with smaller jobs may also arise in the backend implementation. Also, when a smaller job fails or requires reprocessing, the cost will be smaller.
```

Comment:
avoid 's.

It might be good to double check this section with Victor. 

From what I understand it is actually beneficial to process an as large as possible area at once (supported by the amount of memory you have.)

Since then spark really shines and lowers the credit cost.


"Bottlenecks that do not occur with smaller jobs may also arise in the backend implementation. Also, when a smaller job fails or requires reprocessing, the cost will be smaller."

these sentences are vague and confusing